tests/pulser_simulation/test_simulation.py

```diff
@@ -1987,3 +1987,125 @@ def test_detuning_noise():
             np.array([-0.20112646] * (duration) + [0.0] * (duration + 1)),
         )
     )
+
+
+def test_hf_detuning_noise_validation():
```

Comment:
@HGSilveri I wrote tests here because of the `test_detuning_noise()`.
Does it make sense to move this test to `test_noise_model.py` ?